packages/babel-preset-env/src/normalize-options.js

```diff
@@ -24,15 +24,15 @@ const pluginToRegExp = (plugin: any): RegExp => {
   }
 };
 
-const selectPlugins = (regexp: RegExp): Array<string> =>
+const selectPlugins = (regexp: ?RegExp): Array<string> =>
   Array.from(validIncludesAndExcludes).filter(
     item => regexp instanceof RegExp && regexp.test(item),
   );
 
 const flatten = array => [].concat(...array);
 
 const expandIncludesAndExcludes = (
-  plugins: Array<string | RegExp> = [],
+  plugins: Array<string> = [],
```

Comment:
@existentialism Is this right? I thought we supported regexes now?

Reply:
Ack, you're right.

@thymikee can you revert please? Support for this landed in https://github.com/babel/babel/pull/7242